.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
```

Comment:
Oh OK, that's fine then. :)

On Tue, Sep 6, 2016 at 4:21 PM, Sean Stuckless <notifications@github.com>
wrote:

> In .editorconfig
> <https://github.com/google/sagetv/pull/169#discussion_r77736574>:
>
> > @@ -0,0 +1,5 @@
> > +root = true
>
> Actually I did... .editorconfig is a standard configuration file that sets
> the indentation properties for files. It's recognized by eclipse and
> intellij. i can remove it, if you want... but it got added because my
> default spacing for tabs is not 2 and I got tired of having to keep
> reformatting stuff :)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/sagetv/pull/169/files/e9d54339e3e8bd2e1c2c4e907a0b9fd7063f6151#r77736574>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANEIDMKYJHZ6df9OPDPayPgvO-F5UY3eks5qnfVugaJpZM4J2Xlr>
> .
>



-- 
Jeffrey Kardatzke
jkardatzke@google.com
Google, Inc.